### PR TITLE
Bound properties to values

### LINKED ISSUE #184: Test values for different properties
Currently, the properties to test specific values for are defined for all values of a module. This doesn't allow to test values that apply to other properties. That's the reason why `transform` needs to be listed as property to allow testing the value `rotate(calc(15deg + 30deg))` in [Values and Units Level 3](https://css3test.com/#css-values-3).

There are two ways to fix that:
1. Move all property definitions into the values. This means duplicating the properties for each value.
2. Keep the general property definitions but allow individual values to overwrite them.

Personally, I prefer the first option. It means to duplicate the properties but it keeps the test logic and the structure of the tests simple.

Sebastian

### PR DESCRIPTION
**Pull request by SebastianZ**
Moved the properties to test the values for to the individual values. This allows to provide different properties for the values.

Fixes #184

Sebastian

### CODE REVIEW

**Review by LeaVerou — CHANGES_REQUESTED** [review]
This seems extremely repetitive. Why not specify them in the root and override them in individual features if/when needed?

**Comment by SebastianZ** [comment]
> This seems extremely repetitive. Why not specify them in the root and override them in individual features if/when needed?

I've answered that in #184.

Sebastian

**Comment by SebastianZ** [comment]
@LeaVerou Changed. Please have another look!

Sebastian

File: tests/css-values-4.js [hunk 1]
```diff
@@ -238,7 +238,7 @@ export default {
 			},
 			tests: ['clamp(12px, 10 * (1vw + 1vh) / 2, 100px)'],
 		},
-		'calc()': {
+		'math functions in calc()': {
```
Comment (LeaVerou) [inline tests/css-values-4.js:241]: This is also unrelated to this PR

**Comment by LeaVerou** [comment]
Hi there,

I tried to review the diff, but it's very difficult due to the number of unrelated changes included in it. Anyhow, as long as you can still specify properties globally and don't have to repeat them on every single test, I trust you that the changes are fine.

**Review by LeaVerou — APPROVED** [review]

**Comment by LeaVerou** [comment]
(approved so that any pending comments would be flushed, but I stopped leaving comments after a while because they'd be all "this is unrelated, and this, and this, and this"

**Comment by SebastianZ** [comment]
> I tried to review the diff, but it's very difficult due to the number of unrelated changes included in it.

Ok, let's close this in favor of #238 which only includes changes related to this new feature.

I'll make the unrelated changes separately.

Seastian

**Pull request closed without merging**